### PR TITLE
Remove I18n shim

### PR DESCRIPTION
**Pull request by kghbln**
* Originally required to support MediaWiki 1.17.x to 1.22.x

Refs issue https://github.com/SemanticMediaWiki/SemanticMediaWiki/issues/1511

### CODE REVIEW

**Comment by Nikerabbit** [comment]
Yay +1

**Comment by JeroenDeDauw** [comment]
+1

File: load.php [hunk 1]
```diff
@@ -70,7 +70,6 @@ public static function initExtension() {
 
 		// Because of MW 1.19 we need to register message files here
 		$GLOBALS['wgMessagesDirs']['SemanticMediaWiki'] = $GLOBALS['smwgIP'] . 'i18n';
-		$GLOBALS['wgExtensionMessagesFiles']['SemanticMediaWiki'] = $GLOBALS['smwgIP'] . 'languages/SMW_Messages.php';
```
Comment (mwjames) [inline load.php]:
```
There was 1 failure:
1) SMW\Tests\SetupTest::testRegisterMessageFiles with data set #0 ('SemanticMediaWiki', array(array('')))
Failed asserting that false is true.
```

This line is checked in [0], so removing `SemanticMediaWiki` from [0] should make the test pass.

[0] https://github.com/SemanticMediaWiki/SemanticMediaWiki/blob/master/tests/phpunit/includes/SetupTest.php#L279

Comment (kghbln) [inline load.php]: Done! Makes sense. :) Thanks.

**Pull request merged**